### PR TITLE
#161779859 Integrate coveralls and coverage reporter

### PR DESCRIPTION
**Pull request by V-84**
#### What does this PR do?
Integrate coveralls
#### Description of Task to be completed?
- setup coveralls
- setup coverage reporter (Istanbul and integrate with coveralls)
#### How should this be manually tested?
clone repo
- run npm install
- run npm run coverage
#### Any background context you want to provide?
n/a
#### What are the relevant pivotal tracker stories?
[#161779859](https://www.pivotaltracker.com/n/projects/2223728/stories/161779859)

#### Screenshots (if appropriate)
n/a
#### Questions:
n/a

### CODE REVIEW

**Comment by theghostyced** [comment]
Nice work @JohnMadakin. But your commit message seems to be just `chore:`. Try changing it to follow the commit message convention
> https://github.com/andela/engineering-playbook/tree/master/5.%20Developing/Conventions

File: package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "mocha": "mocha --compilers js:babel-register",
```
Comment (efalayi) [inline package.json]: @JohnMadakin is there a reason why `mocha` is added as a script here? Also, I don't think this PR requires `mocha`

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+[![Coverage Status](https://coveralls.io/repos/github/andela/selene-ah-backend/badge.svg?branch=master)](https://coveralls.io/github/andela/selene-ah-backend?branch=develop)
```
Comment (efalayi) [inline README.md]: @JohnMadakin the target branch should be `develop`

Comment (V-84) [reply README.md]: ok, will resolve it now

**Pull request merged**